### ISSUE TITLE
Request Future Episodes/Seasons

### ISSUE THREAD

**Issue by mrhotio** [sct/overseerr#478]
At the moment it's not possible to only request episodes or seasons that are going to take place in the future but are not yet known...something like sonarr's add dialog would be nice.
<img width="538" alt="Schermafbeelding 2020-12-23 om 20 34 45" src="https://user-images.githubusercontent.com/26902309/103032034-5bd7a180-455f-11eb-9148-18ba5e4cd3a7.png">

**Comment by sct** [comment]
Right now it will request everything + the future. To be clear, this is a request to have it so you can _only_ request the future episodes/seasons right?

**Comment by mrhotio** [comment]
yes, only the future

**Comment by mrhotio** [comment]
This issue has been automatically marked as active because it has had recent activity. It will not be closed if further activity occurs. Thank you for your contributions.

**Comment by costaht** [comment]
![robot-robotic](https://user-images.githubusercontent.com/50637431/116026366-90985a00-a620-11eb-89bd-fbd05a263f4b.gif)

**Comment by g0d-m0nky** [comment]
Just commenting to throw my weight behind this request. It would be really useful for when adding shows that you have already seen up until now. Are there any plans to implement it?

**Comment by sadstan** [comment]
I'd love to see this too :)

**Comment by DaClownie** [comment]
Just commenting that is a feature I'm very interested in seeing!

**Comment by nmbrg** [comment]
i came here just to request this (and one other thing). this is one of two things missing that would make overseerr perfect for me.

**Comment by xuatz** [comment]
I'm taking a stab at implementing this feature. Fingers crossed for progress lol.

**Comment by Andystew94** [comment]
Looking forward to this feature

**Comment by cavazos-apps** [comment]
Any hopes of seeing this implemented?

**Comment by nmbrg** [comment]
> Any hopes of seeing this implemented?

wouldnt get my hopes up. its not like they're doing anything at all "recently".
ive jumped ship to jellyseer, you might have a better chance there at getting what you want. works fine with plex.

**Comment by cavazos-apps** [comment]
> > Any hopes of seeing this implemented?
> 
> wouldnt get my hopes up. its not like they're doing anything at all "recently".
> ive jumped ship to jellyseer, you might have a better chance there at getting what you want. works fine with plex.

Yeah I'm using jellyseerr but no one seems to apt to implement. The only active effort was abandoned and the dev deleted the branch for whatever reason. May try an attempt at it myself over there, seems like a no brainer feature.

**Comment by xuatz** [comment]
@cavazos-apps actually I just pushed an updated PR with some questions about the UX a few days ago ^^

maybe you want to chip in the discussion 
https://github.com/sct/overseerr/pull/3093#issuecomment-3121532540

~~Otherwise I'm also totally cool if you want to take a stab at the feature instead~~

oh I think you meant jellyseerr

**Comment by cavazos-apps** [comment]
> @cavazos-apps actually I just pushed an updated PR with some questions about the UX a few days ago ^^
> 
> maybe you want to chip in the discussion 
> https://github.com/sct/overseerr/pull/3093#issuecomment-3121532540
> 
> ~~Otherwise I'm also totally cool if you want to take a stab at the feature instead~~
> 
> oh I think you meant jellyseerr

I personally would like to see it in jellyseerr as that's more maintained but I don't use Jellyfin so I'm happy with whichever implements the feature. May be able to be merged into jellyseerr after.
